docs/core/components/icon.component.md

```diff
@@ -78,6 +80,61 @@ using the `adf:` namespace.
 <adf-icon value="adf:image/gif"></adf-icon>
 ```
 
+### Icon alias mapping
+
+The Icon Alias Mapping feature allows you to provide custom icon value mappings at runtime using the `ICON_ALIAS_MAP_TOKEN` injection token. When an icon value matches a key in the alias map, the component automatically replaces it with the mapped value. This is useful for creating consistent icon conventions across your application without modifying component code.
+
+You can register a single alias map, or use `multi: true` to register multiple alias maps that will be merged together.
+
+**Example with a single alias map:**
+
+```ts
+import { ICON_ALIAS_MAP_TOKEN } from '@alfresco/adf-core';
+
+function getProviders() {
+    return [
+        {
+            provide: ICON_ALIAS_MAP_TOKEN,
+            useValue: {
+                'icon-mock': 'alias-mock',
+                'old-icon': 'new-icon'
+            }
+        }
+    ]
+}
+```
+
+**Example with multiple alias maps using `multi: true`:**
+
+```ts
+function getProviders() {
```

Comment:
not sure this scenario is supported by the component, as it needs an array import